.github/workflows/e2e-hotel.yml

```diff
@@ -42,22 +50,29 @@ jobs:
           username: ${{ secrets.DOCKER_HUB_USERNAME }}
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
       - name: Set up Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@v2
 
+
       - name: Build and push
-        working-directory: benchmarks/hotel-app
-        env:
-          GOPRIVATE_KEY: ${{ secrets.EASE_LAB_ACCESS_KEY }}
```

Comment:
hmm, does it work without that secret?

Reply:
Yes. Why should it not? We should we need a GOPRIVATE_KEY